src/content/fr/fundamentals/performance/rendering/index.md

```diff
@@ -0,0 +1,155 @@
+puisse, de manière incorrecte, apparaître au dessus d'un autre.
+
+Chacune de ces parties du pipeline représente une opportunité d'introduire une
+latence, il est donc important de comprendre les parties du pipeline que votre
+code déclenche.
+
+Parfois, on entend le terme "numérisation" en conjonction avec le rendu.
+C'est parce que le rendu est composé de deux tâches: 1) créer la liste des
+appels de rendu, et 2) remplir avec les pixels.
+
+Ce dernier est appelé "numérisation", et à chaque fois que vous voyez des appels
+de rendu dans DevTools, vous devez penser que cela inclut probablement une
+numérisation. (Dans certaines architectures, la création des appels de rendu et
+la numérisation sont faites dans des threads séparés, mais ce n'est pas quelque
+chose qui est sous le contrôle du développeur.)
+
+Il n'est pas nécessaire pour vous de toucher à chaque étape du pipeline pour
+chaque rendu. En réalité, il y a trois manières pour le pipeline d'avoir un
+impact, *normalement*, sur une certaine trame lorsque vous faites un changement
+visuel, qu'il soit en Javascript, en CSS, ou via les Web Animations:
+
+### 1. JS / CSS > Style > Mise en Page> Rendu > Composition
+
+<img src="images/intro/frame-full.jpg" alt="The full pixel pipeline">
+
+Si vous changez une propriété de "mise en page" lié au changement des propriétés
+géométriques d'un élément, comme sa largeur, sa hauteur, ou sa position avec
+left ou top, le navigateur va devoir vérifier les autres élements et réajuster
+la page. Chacune des parties affectées va devoir être rendue une nouvelle fois,
+et le rendu final va devoir être réagencé avec ces autres élements.
+
+### 2. JS / CSS > Style > Rendu> Composition
+
+<img src="images/intro/frame-no-layout.jpg" alt="The  pixel pipeline without
+layout.">
+
+Si vous faites un changement de "rendu uniquement", comme une image
+d'arrière-plan, la couleur du texte, ou des ombres, en d'autres mots des
+propriétés qui n'affectent pas le rendu d'une page, le navigateur va passer
+l'étape de mise en page, mais il va tout de même faire le rendu.
+
+### 3. JS / CSS > Style > Composition
+
+<img src="images/intro/frame-no-layout-paint.jpg" alt="The pixel pipeline
+without layout or paint.">
+
+Si vous faites un changement qui ne nécessite ni une remise en page, ni un rendu
+supplémentaire, le navigateur passe directement à l'étape de composition.
+
+Cette dernière est la plus économe en termes de ressources et la plus appropriée
+pour des points assez délicats dans le cycle de vie de l'application, comme les
+animations ou le défilement de la page.
+
+Note : si vous souhaitez savoir laquelle de ces trois versions sera déclenché
+par rapport à vos modifications de propriétés CSS, rendez-vous sur  [CSS
+Triggers](https://csstriggers.com).
+Et si vous souhaitez en apprendre plus rapidement sur les performances
+d'animations, dirigez-vous à la section relative au changement des propriétés de
+compositeur uniquement.
```

Comment:
Missing link?
https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count

Reply:
Redirecting to the english version for now.